aii-ks/src/main/perl/ks.pm

```diff
@@ -566,6 +566,9 @@ sub ksuserscript
 
     return unless $config->elementExists ($path);
     my $url = $config->getElement ($path)->getValue;
+    my $server = LOCALHOST;
+    $url =~ s{LOCALHOST}{$server};
```

Comment:
this behaviour should be added to the documentation, at the very least to the comment above the sub
and :+1: for the confusing, altough you could do better with 
```perl
$url =~ s{LOCALHOST}{LOCALHOST}e;
```